src/targets/browser/index.ejs

```diff
@@ -14,6 +14,7 @@
   <% _.forEach(htmlWebpackPlugin.files.css, function(file) { %>
     <link rel="stylesheet" href="<%- file %>">
     <% }); %>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato:400,700,300">
```

Comment:
No way we load it from google!

Reply:
Currently the font is loaded from : 
assets/fonts/fonts.css : https://github.com/cozy/cozy-stack/blob/master/assets/fonts/fonts.css#L5-L12

I don't know if Android supports woff2 format?

Reply:
I did this because that is what https://docs.cozy.io/cozy-ui/react/ does and it works on android.

But I don't know where this css file would be included in the home application.

Looks like woff2 is supported in chrome since 95 https://caniuse.com/?search=WOFF2